example/sketch.js

```diff
@@ -30,6 +34,9 @@ function draw() {
   let range = new Rectangle(mouseX, mouseY, 25, 25);
   rect(range.x, range.y, range.w * 2, range.h * 2);
   let points = qtree.query(range);
+  
+  if (!points) return;
```

Comment:
Why not use brackets here?

Reply:
I'm just used to this, when it's just an "if return" I always do that since it's easily readable and brackets are not necessary imho... It would just add useless lines.
You totally could use them though :)

Reply:
Yeah, magic stuff 😛 I come from the PHP world, even though I know it's possible to do so, my question was just a polite version of _"Please use brackets and don't use implicitly magic syntax"_ , but I'm probably too rigid 🤣

Reply:
Maybe you are ^^ I think as long as simpler syntaxes were invented they can be used. And it really looks clearer to me.

if (!points) {
   return;
}

This is just redundant to me. But you know I'm one of those who like LINQ and lambda expressions, that means everything! ^^

Reply:
This is no longer necessary thanks to #5!

Reply:
Some JS style guides I read recommend using curly brackets in all cases. I personally like the one-liner `if`s without brackets. Especially when there are several in a row.

Maybe because I am more a Ruby programmer than a JS programmer.